ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineGetPayload.java

```diff
@@ -96,7 +96,8 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext request) {
       LOG.debug(
           "Only empty block available for payload {}, waiting for block building to complete",
           payloadId);
-      mergeMiningCoordinator.awaitCurrentBuildCompletion(payloadId, 800);
+      mergeMiningCoordinator.awaitCurrentBuildCompletion(
+          payloadId, mergeMiningCoordinator.getPosBlockFinalizationTimeoutMs());
```

Comment:
minor: the timeout configuration is only used here to be passed back to the `mergeMiningCoordinator`, so it could have been just passed internally, i.e. `mergeMiningCoordinator.awaitCurrentBuildCompletion(payloadId)`, that will simplify the code avoiding to expose the configuration value back and forth